platform-sdk/swirlds-common/src/testFixtures/java/com/swirlds/common/test/fixtures/AssertionUtils.java

```diff
@@ -85,16 +81,16 @@ public static void assertEventuallyTrue(
             if (CompareTo.isGreaterThan(elapsed, maxDuration)) {
                 condition.getAsBoolean();
                 if (cause == null) {
-                    fail(message);
+                    Assertions.fail(message);
```

Comment:
I believe, IntelliJ Idea falls back to the regular imports in case of the refactorings like this one. Could you please return static imports?

Reply:
yeah, must be IntelliJ since I do not touched the code. Will have a look at all PRs. THX